config/routes.rb

```diff
@@ -93,23 +93,23 @@
     end
 
     FormHandler.instance.lettings_forms.each do |_key, form|
-      form.pages.map do |page|
+      form["form"].pages.map do |page|
```

Comment:
`form["form"]` seems a little strange...

Reply:
Yeah I was thinking about this...
For me it felt like if we were to access a specific form by `FormHandler.instance.get_form(_)` that would return just the form object and if we call `FormHandler.instance.forms[0]` that would return the hash with form, type and start year.
We could potentially have something similar where `lettings_forms` returns what it currently does and perhaps `get_lettings_forms` could only return an array of form objects (?) not sure how useful all those methods would be tho

Reply:
I think what feels off is that we're sort of referring to two different object types/data structures (one a super set of the other) and calling them both "form", which might be confusing.